docs/development/decision-records/2024-09-16-proposal-federated-catalog-distribution/README.md

```diff
@@ -0,0 +1,24 @@
+# Proposal for FederatedCatalog with Tractus-X distribution
+
+## Decision
+
+The proposed solution is to have a Federated Catalog as own runnable service capable of crawling all the chosen catalogs and expose that data.
+
+## Rationale
+
+The Federated Catalog aims at providing the aggregation of catalogs from multiple participants in a centralized point to reduce latency.
+
+Choosing a solution that decouples it from the Control Plane (like the one used for the Data Plane) and able to be scalable will future-proof the Federated Catalog as a feature and embraces wider usage.
+
+## Approach
+
+The Federated Catalog running as own instance can be performed either by running respective Docker image in a container or directly running a generated `jar`. 
+
+> In case of aiming at testing locally, the [Readme.md](https://github.com/eclipse-edc/FederatedCatalog/blob/main/README.md) includes a very helpful guide. 
+
+Regarding deployment, the Tractus-Connector Helm charts will be updated to include this service.  
+
+Relevant to highlight a potential challenge of the proposed approach.
+- Having its own instance (that itself contains a cache) may use considerable computing resources (related with storage cost).
```

Comment:
> use considerable computing resources (related with storage cost).

how so? what cost do you anticipate, that wouldn't also be there if the FC ran embedded in the ControlPlane?

Reply:
You're right. The idea was comparing a specific runtime vs embedded in control plane could incur on having more resources used, but should be a minimal difference and I do not have comparing numbers. 
Regarding cost, is a non issue, since same data would be store regardless of solution.
Removed [here](https://github.com/eclipse-tractusx/tractusx-edc/pull/1555/commits/48b1900fa819dbf0ec0f2652fc4fc07ef5cd8525).